jans-cedarling/bindings/cedarling_python/src/authorize/entity_data.rs

```diff
@@ -32,20 +31,30 @@ use serde_pyobject::from_pyobject;
 ///
 /// .. method:: from_dict(cls, value: dict) -> EntityData
 ///     Initialize a new EntityData from a dictionary.
-///     To pass `entity_type` you need to use `type` key.
+///     The dictionary should contain a `cedar_entity_mapping` field with `entity_type` and `id` subfields.
 #[derive(Clone, serde::Deserialize)]
 #[pyclass]
 pub struct EntityData {
+    /// Cedar entity mapping info
+    #[pyo3(set)]
+    #[serde(rename = "cedar_entity_mapping")]
+    pub cedar_mapping: CedarEntityMapping,
+    /// entity attributes
+    #[serde(flatten)]
+    pub attributes: EntityDataAttrs,
+}
+
+/// Cedar entity mapping information
+#[derive(Clone, serde::Deserialize)]
+#[pyclass]
+pub struct CedarEntityMapping {
```

Comment:
There is no constructor defined for `CedarEntityMapping`, so trying to create this object results in this error:
```
   cedar_entity_mapping=CedarEntityMapping(
                         ~~~~~~~~~~~~~~~~~~^
        entity_type="Jans::Application",
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
        id="some_workload_id"
        ^^^^^^^^^^^^^^^^^^^^^
    ),
    ^
TypeError: No constructor defined for CedarEntityMapping
```
Please check `EntityData` type in same file on how to create constructor for type and return PyResult.